css/how_to_name_the_classes.md

```diff
@@ -0,0 +1,118 @@
+# How to name the classes?
```

Comment:
Can we put "CSS" in the title?

Either as a prefix ("CSS: ") or inline ("CSS classes")